returnn/search.py

```diff
@@ -88,7 +90,9 @@ def __init__(
 
         self.out_returnn_config_file = self.output_path("returnn.config")
 
-        self.out_search_file = self.output_path("search_out")
+        self.out_search_file = self.output_path(
+            f"search_out.{output_mode}." + (".gz" if output_gzip else "")
```

Comment:
```suggestion
            f"search_out" + (".{output_mode}.gz" if output_gzip else "")
```

I think this is what I would suggest

Reply:
Ah I see. Ok I did that.